internal/provider/string.go

```diff
@@ -283,9 +361,22 @@ func importString(ctx context.Context, req tfsdk.ImportResourceStateRequest, res
 
 	state.Keepers.ElemType = types.StringType
 
-	if sensitive {
-		state.ID.Value = "none"
+	diags := resp.State.Set(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
 	}
+}
+
+func importPassword(ctx context.Context, req tfsdk.ImportResourceStateRequest, resp *tfsdk.ImportResourceStateResponse) {
```

Comment:
Similar to the note on the sdk/v2 version of this, since the new attribute is only handled once on creation, it will also need to be handled during import. It _may_ be desirable to optionally include it in the import ID so folks can have a stable value when doing `terraform state rm` and `terraform import`, although that does introduce yet more complexity. Maybe that can be punted on until there is a specific ask.

Reply:
Alternatively, could the `bcrypt_hash` value be just computed during import?

After all what matters about it is not strictly having a specific value, but that it's stable once generated.

I'd leave out the option to import it with the ID only if it comes up again, and hopefully deferred to when "import-from-config" is a thing.

Reply:
I've updated `importPassword` to handle `bcrypt_hash` during import. 

I'm inclined to defer handling of a passed `bcrypt_hash` during import until/unless it is specifically requested.